src/vanilla/utils/atomFamily.ts

```diff
@@ -4,6 +4,7 @@ type ShouldRemove<Param> = (createdAt: number, param: Param) => boolean
 
 export interface AtomFamily<Param, AtomType> {
   (param: Param): AtomType
+  has(atom: AtomType): boolean
```

Comment:
Thanks. This is exactly what I imagined including the implementation.

I'm a bit hesitant with this new api, though.
- I don't see other use cases than jotai-scope
- I would like to see another solution "injecting callback" and compare with it

```suggestion
  onCreate(callback: (param: Param, atom: AtomType) => void): void
  getParams(): Iterator<Param>
```

Reply:
> "injecting callback"

It requires `onRemove` too.